src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -303,11 +304,18 @@ export class EditDataGridPanel extends GridParentComponent {
 			return;
 		}
 
+		// get the cell we have just immediately clicked if its different as the active cell will be reset to the previous cell by slickgrid if rerendered.
+		if (row !== this.currentCell.row || column !== this.currentCell.column) {
```

Comment:
Do you actually need this check or could you just update it every time? It doesn't really matter if it's the same cell since the end value will be the same regardless